e2e/testcases/admission_test.go

```diff
@@ -141,55 +148,82 @@ metadata:
 }
 
 func TestDisableWebhookConfigurationUpdate(t *testing.T) {
-	webhook := []byte(`
-apiVersion: admissionregistration.k8s.io/v1
-kind: ValidatingWebhookConfiguration
-metadata:
-  annotations:
-    configsync.gke.io/webhook-configuration-update: disabled
-  name: admission-webhook.configsync.gke.io
-  labels:
-    configmanagement.gke.io/system: "true"
-    configmanagement.gke.io/arch: "csmr"
-`)
-
 	nt := nomostest.New(t, nomostesting.DriftControl)
 
+	// Test starts with Admission Webhook already installed
+	nomostest.WaitForWebhookReadiness(nt)
+
 	nt.Must(nt.RootRepos[configsync.RootSyncName].Add("acme/namespaces/hello/ns.yaml", fake.NamespaceObject("hello")))
-	nt.Must(nt.RootRepos[configsync.RootSyncName].CommitAndPush("add Namespace"))
+	nt.Must(nt.RootRepos[configsync.RootSyncName].Add("acme/namespaces/world/ns.yaml", fake.NamespaceObject("world")))
```

Comment:
Do we need a separate Namespace? When the `hello` Namespace is deleted without drift prevention, it should be recreated by the remediator.

Reply:
Reverted to `hello` only.